api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -54,11 +54,31 @@ message UpstreamTlsContext {
   google.protobuf.UInt32Value max_session_keys = 4;
 }
 
-// [#next-free-field: 8]
+// [#next-free-field: 9]
 message DownstreamTlsContext {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.api.v2.auth.DownstreamTlsContext";
 
+  enum OcspStaplePolicy {
+    // OCSP responses are optional. Invalid or expired OCSP
+    // responses are rejected at config time. If an OCSP
+    // response expires before a config update, it is no
+    // longer stapled.
+    SKIP_STAPLING_IF_EXPIRED = 0;
+
+    // OCSP responses are required on all certificates for a
+    // configuration to be deemed valid. If an OCSP response
+    // expires before a config update, subsequent TLS
+    // handshakes for that certificate are aborted.
+    STAPLING_REQUIRED = 1;
+
+    // OCSP responses are optional. Invalid or expired OCSP
+    // responses are rejected at config time. If an OCSP
+    // response expires before a config update, subsequent
+    // TLS handshakes for that certificate are aborted.
+    REJECT_CONNECTION_ON_EXPIRED = 2;
```

Comment:
I don't understand the difference betwen 1 and 2. Can you clarify how the behavior is different?

Reply:
Hopefully clarified in the above discussion on renaming the policies